ios-base/DroidKaigi 2020/Session/Detail/SessionDetailViewController.swift

```diff
@@ -0,0 +1,204 @@
+import ios_combined
+import Nuke
+import RxCocoa
+import RxSwift
+import UIKit
+
+final class SessionDetailViewController: UIViewController {
+    var session: Session!
+
+    // MARK: Private
+
+    private let disposeBag = DisposeBag()
+
+    // MARK: Outlets
+
+    @IBOutlet private weak var headerSection: UIView!
+    @IBOutlet private weak var titleLabel: UILabel!
+    @IBOutlet private weak var dateLabel: UILabel!
```

Comment:
Where is it defined?
Is it able to use in IBs?

Reply:
It is defined in `ApplicationScheme`.
To use the color with `ApplicationScheme.shared.colorScheme.primaryColor`.

I think it can't use in IBs 🙏

Reply:
How about using Color Assets?

In ApplicationScheme, it can initialize like next.

```swift
scheme.primaryColor = UIColor(named: "Primary")!
scheme.onPrimaryColor = UIColor(named: "OnPrimary")!
scheme.secondaryColor = UIColor(named: "Secondary")!
scheme.surfaceColor = UIColor(named: "Surface")!
scheme.onSurfaceColor = UIColor(named: "OnSurface")!
```

I don't know if it will support Dark Mode, but if does, I think it's easier to keep it here.

Reply:
It's good! 😄
Would you apply this one in this PR or another PR?

Reply:
I did it in a97d0111 and 0f1e26ee!

Reply:
It's good! Thanks! 😄